docs/guide/data_migration/importing_data.md

```diff
@@ -0,0 +1,171 @@
+# Importing data
+
+To import data from YAML migration files into Repository, run the `ibexa:migrations:migrate` command.
+
+You can use the `ibexa:migrations:import` command to automatically place migration files in the correct folder.
```

Comment:
`import` command does indeed **place** the files in the migrations folder.

Command cannot be run without parameters (it will throw an error if tried), but `--from-stdin` option can be used to instruct it to read the file from the standard input instead (which means some other script can generate the file and pipe it into the command: `generate.sh > php bin/console ibexa:migrations:import --from-stdin` (haven't tested the exact instructions, but you get the idea)).

And you understand the command correctly. It's supposed to be used to pick YAML migration files from elsewhere and place them in the right directory. It's primary reason for existence is that this directory can be configured to be elsewhere (if not right now, it is supposed to be configurable).